python/src/gi-stubs/repository/Ygg.pyi

```diff
@@ -0,0 +1,160 @@
+from enum import IntEnum
+from typing import Any
+from typing import Callable
+from typing import Optional
+from typing import TypeAlias
```

Comment:
nitpick: what about a single `from typing import ...` line, as done a couple of lines later and also in the example worker?